### PR TITLE
Fix lint, format and fix tests

### PR DESCRIPTION
**Pull request by piperchester**
## Description

Small PR to fix `yarn lint`, `yarn format`, and a couple tests.

## Todo

- [x] PR has a meaningful title?
- [x] Lint passes?
- [x] Ran `yarn format` in frontend?

### CODE REVIEW

File: frontend/pages/chat/[chatUUID].js [hunk 1]
```diff
@@ -29,8 +29,8 @@ export default function Chat({
     nextLink: nextLink,
```
Comment (piperchester) [inline frontend/pages/chat/[chatUUID].js:29]: Just fixing `yarn format` on this file.

File: frontend/pages/donate.js [hunk 2]
```diff
@@ -298,13 +298,18 @@ export default function Donate({ goal_name, goal_amount, current_amount }) {
           text={
```
Comment (piperchester) [inline frontend/pages/donate.js:298]: Just fixing `yarn format` on this file.

File: frontend/pages/manageProjectMembers/[projectUrl].js [hunk 3]
```diff
@@ -35,13 +35,19 @@ export default function manageProjectMembers({
   );
```
Comment (piperchester) [inline frontend/pages/manageProjectMembers/[projectUrl].js:35]: Just fixing `yarn format` on this file.

File: frontend/pages/profiles/[profileUrl].js [hunk 4]
```diff
@@ -155,10 +155,14 @@ function ProfileLayout({
     Router.push("/chat/" + chat.chat_uuid + "/");
```
Comment (piperchester) [inline frontend/pages/profiles/[profileUrl].js:155]: Just fixing `yarn format` on this file.

File: frontend/public/data/organization_info_metadata.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ import DescriptionIcon from "@material-ui/icons/Description";
 import SchoolIcon from "@material-ui/icons/School";
```
Comment (piperchester) [inline frontend/public/data/organization_info_metadata.js:3]: Just fixing `yarn format` on this file.

File: frontend/src/components/account/AccountPage.js [hunk 6]
```diff
@@ -116,77 +116,76 @@ export default function AccountPage({
...
 
   const displayAccountInfo = (info) =>
-    Object.keys(info).sort((a, b) => {
-      a = getFullInfoElement(infoMetadata, a, info[a])
-      b = getFullInfoElement(infoMetadata, b, info[b])
-      return b?.weight - a?.weight
-    }).map((key, index) => {      
-      if (info[key]) {
-        const i = getFullInfoElement(infoMetadata, key, info[key]);
-        const Icon = i.icon
```
Comment (piperchester) [inline frontend/src/components/account/AccountPage.js:126]: I also remove `Icon` as it was unused to fix lint.

File: frontend/src/components/communication/chat/ChatContent.js [hunk 7]
```diff
@@ -1,11 +1,11 @@
-import React, { useContext } from "react";
-import MiniProfilePreview from "../../profile/MiniProfilePreview";
+import React from "react";
 import { Button, IconButton, TextField, makeStyles, Tooltip } from "@material-ui/core";
-import Messages from "./Messages";
-import UserContext from "../../context/UserContext";
```
Comment (piperchester) [inline frontend/src/components/communication/chat/ChatContent.js:5]: `useContext` and `userContext` were unused. Fixed `format`.

File: frontend/test/components/general/header.test.js [hunk 8]
```diff
@@ -12,6 +13,6 @@ describe("Header Component", () => {
       </ThemeProvider>
     );
     expect(wrapper.find("img")).toHaveLength(1);
-    expect(wrapper.find("img").prop("alt")).toEqual("Climate Connect");
+    expect(wrapper.find("img").prop("alt")).toEqual("Climate Connect logo");
```
Comment (piperchester) [inline frontend/test/components/general/header.test.js:16]: Fixed this test.

File: frontend/test/utils/index.test.js [hunk 9]
```diff
@@ -14,11 +14,14 @@ describe.only("utils", () => {
...
-      const fixedElapsedMs = msSinceEpochFromFixedDate - msSinceEpochFromYearAgo;
+      const fixedStartDateMs = new Date("2020-01-10T22:17:38.387858Z").getTime();
+      const fixedNow = () => {
+        return new Date("2021-01-29T21:34:45.246Z").getTime();
+      };
 
-      expect(yearAndDayFormatter(1, "year", null, fixedElapsedMs)).toBe("1 year and 20 days ago");
+      expect(yearAndDayFormatter(1, "year", null, fixedStartDateMs, () => {}, fixedNow)).toBe(
+        "1 year and 20 days ago"
+      );
```
Comment (piperchester) [inline frontend/test/utils/index.test.js:24]: Fixed this test.

File: frontend/src/components/communication/chat/ChatHeader.js [hunk 10]
```diff
@@ -4,7 +4,7 @@ import KeyboardArrowLeftIcon from "@material-ui/icons/KeyboardArrowLeft";
 import ChatTitle from "../../communication/chat/ChatTitle";
```
Comment (piperchester) [inline frontend/src/components/communication/chat/ChatHeader.js:4]: Just fixing `yarn format` on this file.

**Comment by piperchester** [comment]
@climateconnect/web-frontend I just created this GitHub team as a group to be able to [request review](https://github.blog/2017-06-28-requesting-reviews-from-repository-teams/) from multiple contributors, but it doesn't look like we can which is puzzling? Maybe our instance level?

**Review by positiveimpact — APPROVED** [review]
LGTM! Thanks for aligning the code with with tests, format and lint again, great job!

**Pull request merged**